### ISSUE TITLE
Killing a minimized/paused player

### ISSUE THREAD

**Issue by ghost** [LVPlayground/playground#94]
It's possible to kill a paused player. Tested with various weapons, such as the sniper rifle, sawnoff shotgun, etc. I'm wondering if there's changes that we can make to change this behavior. I think the vast majority of players will pause because they're working on something in the background, rather than, for example, pausing to avoid death -- and it's the vast majority of people that we should be catering for. 

If it turns out this isn't something we can change - fine.

If it is something we can change, perhaps we should start a poll in the crew or public forums to see which option would be preferred. 

https://youtu.be/N4Nj4hw7myM

**Comment by mblokker** [comment]
We used to have that, but only with sniper headshots. That was fixed (https://github.com/LVPlayground/playground/blob/master/pawn/Resources/Callbacks/OnPlayer/OnPlayerGiveDamage.pwn#L16) a while ago (Trac-era).

This seems new to me, but I think we can fix this using http://wiki.sa-mp.com/wiki/OnPlayerWeaponShot. By checking the hitType (person) and hitId (personId), we could check if the player is minimized, and then do simply: if (Player(personId)->isMinimized() == true) return 0.

I think we don't need a poll for this. When pauzed, you shouldn't be able to take damage. If you abuse pauze, crew should punish that.
